### DISCUSSION TITLE
noob problems (import modules)

### DISCUSSION THREAD

**Discussion by Gfernandezv** [time4tea/gopro-dashboard-overlay#164] [Q&A]
I’m just trying to install GoPro overlay, but I have the following error every time that I try to create a video

  File "C:\Users\germa\gopro-dashboard-overlay-main\bin\gopro-dashboard.py", line 11, in <module>
    from gopro_overlay import timeseries_process, progress_frames, gpmd_filters

Maybe I just lost a module, but I don’t know which one. I tried to run it in Python for Windows (11) and in Anaconda.
(FFmpeg ok)

greetings.

**Comment by time4tea** [comment]
Hi there - Thanks for trying the software! Please do try to  follow the windows instructions exactly ( https://github.com/time4tea/gopro-dashboard-overlay#installation-on-windows ) - If you get an error, please post the entire output and I'll see if I can help you figure out the problem. Cheers!
